pkg/shed/field_string_test.go

```diff
@@ -37,7 +37,7 @@ func TestStringField(t *testing.T) {
 
 	t.Run("get empty", func(t *testing.T) {
 		got, err := simpleString.Get()
-		if err != nil {
+		if err == nil {
 			t.Fatal(err)
```

Comment:
you'd probably want to write a custom error message here for the test, since you're failing the test with a `nil` value.

Reply:
I'm merging the PR as is, you can piggy-back this change on a later PR

Reply:
Thanks @acud